code/modules/research/techweb/all_nodes.dm

```diff
@@ -162,7 +162,7 @@
 	design_ids = list("solarcontrol", "recharger", "powermonitor", "rped", "pacman", "adv_capacitor", "adv_scanning", "emitter", "high_cell", "adv_matter_bin", "scanner_gate",
 	"atmosalerts", "atmos_control", "recycler", "autolathe", "high_micro_laser", "nano_mani", "mesons", "thermomachine", "rad_collector", "machine_igniter", "mass_driver", "tesla_coil", "grounding_rod",
 	"apc_control", "cell_charger", "power control", "airlock_board", "firelock_board", "aac_electronics", "airalarm_electronics", "firealarm_electronics", "cell_charger", "stack_console", "stack_machine",
-	"oxygen_tank", "plasma_tank", "emergency_oxygen", "emergency_oxygen_engi", "crystallizer", "plasmaman_tank_belt", "electrolyzer", "plasmaman_tank", "antivirus2", "liquid_output_pump")//MONKESTATION CHANGE
+	"oxygen_tank", "plasma_tank", "emergency_oxygen", "emergency_oxygen_engi", "crystallizer", "plasmaman_tank_belt", "electrolyzer", "plasmaman_tank", "antivirus2", "liquid_output_pump", "evaporation_machine")//MONKESTATION CHANGE
```

Comment:
I'm thinking it should be moved to Cryostasis Technology (Tier 3), or Experimental Tools (Tier 3).